### PR TITLE
Add Provisioning HOC to the webapp

### PR DESCRIPTION
**Pull request by aidenkeating**

### CODE REVIEW

File: src/components/provisioning/provisioning.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      return svc.status.conditions[0].status === 'True';
+    }
+    return false;
+  }
+
+  // NOTE: All styling is inline as it'll all be removed anyway.
+  loadingScreen(services) {
+    const provisionProgress = Provisioning.getMiddlwareServiceProgress(services);
+    return (
+      <div style={{marginLeft: 'auto', marginRight: 'auto', display: 'block', textAlign: 'center', maxWidth: '800px', marginTop: '150px'}}>
```
Comment (mindreeper2420) [inline src/components/provisioning/provisioning.js]: @aidenkeating can you create a class for this and attach these attributes to that? That will make it easier for future revisions, to just have to change a CSS file. Thanks!

**Comment by aidenkeating** [comment]
@dlabaj @mindreeper2420 Mind taking a look?

File: src/styles/application/_provisioning.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  position: absolute;
+  top: 10px;
+  right: 50px;
+  background-image: url('../../img/RedHatLogo.png')
+}
+
+.loadingscreen-throbber {
+  display: inline-block;
+  position: fixed;
+  top: 0;
```
Comment (mindreeper2420) [inline src/styles/application/_provisioning.scss]: change to 10%

**Comment by aidenkeating** [comment]
@mindreeper2420 Thanks for the review! Updated.

File: src/styles/application/_provisioning.scss [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+.loadingscreen-progressbar .progress-bar {
+  background-color: $pf-color-white;
+  background-image: none;  
+}
+
+.loadingscreen-progress-label {
+  display: inline-block;
+  position: fixed;
+  top: 55%;
```
Comment (mindreeper2420) [inline src/styles/application/_provisioning.scss]: my bad - this should have been 50% (in order for it to line up with the progress bar).

Comment (aidenkeating) [reply src/styles/application/_provisioning.scss]: @mindreeper2420 Makes sense, updated

**Pull request merged**